templates/app/models/index.hbs

```diff
@@ -1,16 +1,21 @@
 {{#if isMongoDB}}
 const mongoose = require('mongoose');
-{{#each models as |model|}}
-const {{model.modelName}} = require('./{{model.modelFile}}');
-{{/each}}
+const requireAll = require('require-all');
 
-mongoose.connect(process.env.DATABASE_URL, { useNewUrlParser: true, useUnifiedTopology: true });
+const models = {};
 
-const models = {
-{{#each models as |model|}}
-  {{model.modelName}},
-{{/each}}
-};
+mongoose.connect(process.env.DATABASE_URL, { useNewUrlParser: true });
+
+const modelsList = requireAll({
+  dirname:  __dirname,
+  filter: `^(?!index\.js$).+`,
```

Comment:
Following what we have in `forest-express` `requireAll`, maybe:
```javascript
  excludeDirs: /^__tests__$/,
  filter: (fileName) => isJavascriptFileName(fileName) && !isTestFileName(fileName),
```
could be a good improvement.

Reply:
As discussed together, this will be handled in the future.